<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableContent</class>
 <widget class="QWidget" name="tableContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>773</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>770</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="force_GroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>60</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>210</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>220</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="force_Label">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>180</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>力量评定历史数据</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QChartView" name="force_GraphicsView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>570</width>
          <height>180</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="image_Label">
        <property name="geometry">
         <rect>
          <x>537</x>
          <y>20</y>
          <width>72</width>
          <height>46</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="scope_GroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>210</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>220</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="scope_Label">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>180</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>关节活动度历史数据</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QChartView" name="scope_GraphicsView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>570</width>
          <height>180</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="time_GroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>210</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>220</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="time_Label">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>180</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>运动时间历史数据</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QChartView" name="time_GraphicsView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>550</width>
          <height>180</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
